pages/sponsor/prospectus.vue

```diff
@@ -1,104 +1,232 @@
+                <p>{{ $t('statsTitle') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc ml-4">
+                    <li
+                        v-for="(item, i) in $t('stats')"
+                        :key="`prospectus_stats_${i}`"
+                    >
+                        {{ item }}
+                    </li>
+                </ul>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
         <!-- important dates -->
-        <h2>{{ $t('importantDates.0') }}</h2>
-        <i18n path="importantDates.1" tag="p"></i18n>
-        <ul class="list-disc">
-            <i18n
-                v-for="(date, i) in $t('importantDates.2')"
-                :key="`prospectus_dates_${i}`"
-                :path="`importantDates.2.${i}`"
-                tag="li"
-            >
-                <template #AoE>
-                    <ext-link
-                        href="https://www.timeanddate.com/worldclock/converter.html?iso=20210427T115900&p1=tz_aoe&p2=241&p3=1440"
-                        highlight
-                        >AoE</ext-link
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('importantDatesTitle') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc ml-4">
+                    <i18n
+                        v-for="(date, i) in $t('importantDates')"
+                        :key="`prospectus_dates_${i}`"
+                        :path="`importantDates.${i}`"
+                        tag="li"
                     >
-                </template>
-            </i18n>
-        </ul>
-
-        <!-- sponsor package -->
-        <h2>{{ $t('package-title') }}</h2>
-        <div
-            v-for="(level, i) in sponsorLevels"
-            :key="`prospectus_package_${i}`"
-        >
-            <h3 :id="level">
-                <b>{{ $t(`package-${level}.0`) }}</b>
-            </h3>
-            <ul class="list-disc">
-                <li
-                    v-for="(item, j) in $t(`package-${level}.1`)"
-                    :key="`prospectus_package_${i}_detail_${j}`"
+                        <template #AoE>
+                            <ext-link
+                                href="https://www.timeanddate.com/worldclock/converter.html?iso=20210427T115900&p1=tz_aoe&p2=241&p3=1440"
+                                highlight
+                                >AoE</ext-link
+                            >
+                        </template>
+                    </i18n>
+                </ul>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <!-- sponsor package mobile -->
+        <p class="section-title text-sm md:text-lg lg:hidden">
+            {{ $t('package-title.0') }}
+        </p>
+        <div class="grid gap-x-3 mt-8 lg:hidden sponsor-packages">
+            <template v-for="row in Array.from(Array(15).keys())">
```

Comment:
Need to explain why it's 15 (and 10 for pc view) in the function, either make it `(length of title array)*(length of displayed item per row)` or simply add a comment for the explanation.